auto_tests/crypto_test.c

```diff
@@ -3,6 +3,7 @@
 #endif
 
 #include "../toxcore/net_crypto.h"
+#include "../toxcore/misc_tools.h"
```

Comment:
`m` < `n` - sort includes. Also, while you're at it, add an empty line after the local includes block.